Makefile

```diff
@@ -0,0 +1,8 @@
+
+all: runKompile runJavac
+
+runKompile: semantics/llvm-syntax.k semantics/llvm-syntax-kompiled
+	kompile semantics/llvm-syntax.k --backend java
```

Comment:
When I was using the LLVM syntax, I was using a version of K that didn't have Java backend as default.